mesheryctl/cmd/vars.go

```diff
@@ -24,9 +24,10 @@ import (
 )
 
 const (
-	url                    = "http://localhost:9081"
-	fileURL                = "https://raw.githubusercontent.com/layer5io/meshery/master/docker-compose.yaml"
-	dockerComposeBinaryURL = "https://github.com/docker/compose/releases/download/1.24.1/docker-compose"
+	url     = "http://localhost:9081"
+	fileURL = "https://raw.githubusercontent.com/layer5io/meshery/master/docker-compose.yaml"
+	// dockerComposeBinaryURL = "https://github.com/docker/compose/releases/download/1.24.1/docker-compose"
```

Comment:
We may want to remove this commented line as I don't know that it provides much value as a historical reference.

Reply:
@gunjanmimo I might be missing it, but was this issue fixed (was the commented line removed)? GitHub makes this hard for me to tell - 
![Screen Shot 2019-10-03 at 6 53 01 AM](https://user-images.githubusercontent.com/7570704/66124611-757f7880-e5aa-11e9-9c08-da5f9e01745e.png)

Reply:
@anandrkskd will you confirm?

Reply:
Yes, sure sir I will update if it's not done yet.